lib/component.js

```diff
@@ -505,10 +507,34 @@ class Component extends WebComponent {
         attrValue = this.getJSONAttribute(attr);
       }
 
-      this.attrs[attr] = attrValue;
+      this._attrs[attr] = attrValue;
+
+      if (attrSchema.deprecatedMsg) {
+        console.warn(`${(this.nodeName || ``).toLowerCase()}: attr '${attr}' is deprecated. ${attrSchema.deprecatedMsg}`);
```

Comment:
Component already implements `toString`, so this should work:
```suggestion
        console.warn(`${this}: attr '${attr}' is deprecated. ${attrSchema.deprecatedMsg}`);
```

Reply:
That kinda felt too noisy with `SIMPLE-APP#cjvogk9f800003h9hr8wdljcu`. What are your thoughts if we changed the .toString to be simply the lowerCase version of nodeName ?

Reply:
I get the feeling you mainly care about lowercasing. Feel free to lowercase. I don't want to remove the instance ID, as that is actually very useful for debugging.

Reply:
I did mind the long panelID which is gibberish but not a biggie. used ${this} as you suggested.